pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1563,6 +1564,17 @@ func (n *NetworkProvider) getNonOverlappingDefaultCIDR(ctx context.Context) (*ne
 		"10.255.255.255/8":  fmt.Sprintf("10.0.0.0/%s", defaultCIDRMask),
 		"172.31.255.255/12": fmt.Sprintf("172.16.0.0/%s", defaultCIDRMask),
 	}
+	podCIDR, err := n.getPodCIDR(ctx, logger)
+	logger.Info("podCidr: ", podCIDR)
```

Comment:
NP, is this log necessary as an `info`

Reply:
no sorry that was for testing while the the pr is in wip